Makefile

```diff
@@ -10,8 +10,18 @@ examples:
 	cargo test --example json_to_edn --features "json"
 	cargo test --example edn_to_json --features "json"
 	cargo run --example async
+	cargo run --example struct_from_str --no-default-features
 
-doc-tests:
+tests:
+	cargo test --all-features --no-fail-fast
+	cargo test --no-default-features --no-fail-fast
 	cargo test --doc
 
-test: unit integration examples doc-tests
+clippy:
```

Comment:
With rust 1.74 we can probably move this configurations to something like this:
https://github.com/rewin123/space_editor/blob/main/Cargo.toml#L66

WDYT?

Reply:
https://github.com/naomijub/edn-rs/blob/master/Cargo.toml#L13
We actually already have this. This is just an explicit check to match CI. Once CI runners and we're at rust 1.76 we can clean both up.